src/subscription_create/subscription_create/http_adapter.py

```diff
@@ -0,0 +1,25 @@
+from requests.adapters import HTTPAdapter, Retry
```

Comment:
This is a duplicate with src/email_care_provider/email_care_provider/http_adapter.py.  Is that intentional or a mistake?  Feels like we've ended up with a packaging problem if we're forced into it.

Reply:
Yea, we wanted to have lambda images as small as possible, so there will be duplication.

Another route would be separate `common` package but that's something for the future.

Reply:
Ok. Let's go with it for the moment; can you write up an issue for it so that the task of pulling out a `common` doesn't get lost?

Reply:
https://thepsc.atlassian.net/browse/HANS-371